### ISSUE TITLE
macOS binary release seems to be broken

### ISSUE THREAD

**Issue by vprithvi** [jaegertracing/jaeger#2101]
The macOS link on https://www.jaegertracing.io/download/ results in an error
```
<Error>
<Code>NoSuchKey</Code>
<Message>The specified key does not exist.</Message>
<Key>56342508/ed8f3c00-570d-11ea-8612-117a4f084963</Key>
<RequestId>2FBEFE20C583E226</RequestId>
<HostId>
pkFkoVjSkDx2ZNhVgAe+JVFSCmj2dOFxkqxRUJPCgZYeI9xA0pJrn1j7zuLtrPZBZYTSw7AsVec=
</HostId>
</Error>
```

It also appears that the macOS binary is legitimately missing under assets for the release: https://github.com/jaegertracing/jaeger/releases/tag/v1.17.0

## Proposal - what do you suggest to solve the problem or improve the existing situation?

We could fix the release process. 

The most recent macOS binary is `jaeger-1.16.0-darwin-amd64.tar.gz`. 

In `1.17.0`, we see `jaeger-1.17.0-linux-s390x.tar.gz` instead of the `darwin` binary

**Comment by yurishkuro** [comment]
from the build logs:
https://travis-ci.org/jaegertracing/jaeger/jobs/654542767#L2055
```
Deploying to repo: jaegertracing/jaeger
Current tag is: v1.17.0
jaeger-1.17.0-linux-s390x.tar.gz already exists, skipping.
jaeger-1.17.0-windows-amd64.tar.gz already exists, skipping.
jaeger-1.17.0-linux-amd64.tar.gz already exists, skipping.
jaeger-1.17.0-darwin-amd64.tar.gz already exists, skipping.
Setting target_commitish to 00985cadeb757886ac158f10e4b8229067797b65
```

**Comment by yurishkuro** [comment]
It's odd. When I quered for assets via API, e.g. `curl -H "Authorization: token $GITHUB_TOKEN" https://api.github.com/repos/jaegertracing/jaeger/releases/23939879/assets`, I saw the darwin tarball in the list, along with the other 3 builds, but the download link fails with `NoSuchKey` error. At the same time, the asset showed 60+ downloads. Something is broken on GitHub, it appears.

I deleted the asset and uploaded it manually (docs: https://developer.github.com/v3/repos/releases/#upload-a-release-asset):
```
curl \
    -H "Authorization: token $GITHUB_TOKEN" \
    -H "Content-Type: $(file -b --mime-type $FILE)" \
    --data-binary @$FILE \
    -v -X POST \
    "https://uploads.github.com/repos/jaegertracing/jaeger/releases/23939879/assets?name=$(basename $FILE)"
```
The download now works. I don't think anything was specifically broken in our release process.
